docs/api/rbac/rbac-example.mdx

```diff
@@ -0,0 +1,81 @@
+---
+sidebar_position: 2
+title: Create RBAC policy
+---
+
+### Example #1
+
+#### We have a shared todo app that allows users to create, read, update, toggle and delete tasks.
+
+First, lets define our roles resources and permissions.
+
+**Roles** - Admin and Operator.<br/>
+**Resource** - Task with create, read, update, toggle and delete actions.<br/>
+**Permissions** - Admin can do everything and Operator can only read and toggle tasks.<br/>
+
+Now let's create a Roles and Resources and Role Permissions through the API.<br/>
+But before we need to get our `API_SECRET_KEY` from the dashboard and get current `project_id` and `env_id`<br/>
```

Comment:
I think because this is literally in every file, let's just have a section in the docs that talks to this and refers back to this guide in all the others?